evap/evaluation/models.py

```diff
@@ -1028,19 +1028,23 @@ def __render_string(cls, text, dictionary):
         return Template(text).render(Context(dictionary, autoescape=False))
 
     @classmethod
-    def send_to_users_in_courses(self, template, courses, recipient_groups):
+    def send_to_users_in_courses(self, template, courses, recipient_groups, cc):
         user_course_map = {}
         for course in courses:
             responsible = course.responsible_contributor
-            for user in self.recipient_list_for_course(course, recipient_groups):
-                if user not in responsible.cc_users.all() and user not in responsible.delegates.all():
-                    user_course_map.setdefault(user, []).append(course)
+            all_recipients = self.recipient_list_for_course(course, recipient_groups)
+            cc_recipients = []
+            if responsible in all_recipients and cc == True:
+                cc_recipients.extend(responsible.cc_users.all())
+                cc_recipients.extend(responsible.delegates.all())
+            recipients = [recipient for recipient in all_recipients if recipient not in cc_recipients]
```

Comment:
why does it break? if a user is a delegate of the responsible person of a course, this course will not be added to the list for the delegate, that's right. but other courses where this person is a recipient will be added to this users course list as usual - or am i wrong here?